javalib/src/main/resources/scala-native/z.c

```diff
@@ -1,3 +1,5 @@
+#ifdef SCALANATIVE_LINK_Z
```

Comment:
I don't now how I missed this one also, but now we have bindings defined in different project then their implementation. They shall be defined in the same project. I've raised a question how to deal with them in #3555

Reply:
I'm going to temporaly move them back to nativelib

Reply:
I can do a followup PR. I moved the `z.c` to the `javalib` to follow the bindings with the Scala code that calls them but missed it on rebase,

Reply:
No need for that I'll recover that in rebase of Armans PR